game-core/src/main/java/games/strategy/triplea/delegate/battle/MustFightBattle.java

```diff
@@ -106,6 +106,8 @@
 
   private static final long serialVersionUID = 5879502298361231540L;
 
+  private static final long MAX_INFINITE_ROUNDS = 10000;
```

Comment:
What do you think of naming this `MAX_ROUNDS` instead?
"max infinite" does not quite grok well for me.